service/src/main/java/bio/terra/catalog/api/PublicApiController.java

```diff
@@ -44,4 +69,10 @@ public ResponseEntity<VersionProperties> getVersion() {
   public String index() {
     return "redirect:swagger-ui.html";
   }
+
+  @RequestMapping(value = "/swagger-ui.html")
```

Comment:
```suggestion
  @RequestMapping(value = "/swagger-ui.html", method = RequestMethod.GET)
```

I think this will resolve the SonarQube complaint.